### PR TITLE
Added darwin-op_team_1 controller to the project and updated it

### PR DESCRIPTION
**Pull request by DavidMansolino**
Added darwin-op_team_1 controller to the project and updated it

### CODE REVIEW

**Comment by fabienrohrer** [comment]
Thank you

File: projects/contests/robotstadium/controllers/darwin-op_team_1/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+LIBRARIES = "$(RESOURCES_PATH)/lib/managers.a"
+LINK_DEPENDENCIES = $(RESOURCES_PATH)/lib/managers.a
+CPP_SOURCES = darwin-op_team_1.cpp FieldPlayer.cpp GoalKeeper.cpp Player.cpp
+JAR_FILE=darwin-op_team.jar
+
+### Do not modify: this includes Webots global Makefile.include
+space :=
+space +=
+WEBOTS_HOME_PATH=$(subst $(space),\ ,$(strip $(subst \,/,$(WEBOTS_HOME))))
+include $(WEBOTS_HOME_PATH)/resources/projects/default/controllers/Makefile.include
```
Comment (fabienrohrer) [inline projects/contests/robotstadium/controllers/darwin-op_team_1/Makefile:53]: Oops. this should be:
include $(WEBOTS_HOME_PATH)/resources/Makefile.include

Could you do another PR resolving that?
Thank you in advance

Comment (DavidMansolino) [inline projects/contests/robotstadium/controllers/darwin-op_team_1/Makefile:53]: Oh, sorry,
I forgot this,
No problem, a do it now.

File: projects/contests/robotstadium/controllers/darwin-op_team_1/FieldPlayer.cpp [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      // compute the direction of the head
+      // the head move at maximum by 0.01 [rad] at each time step
+      x  = 0.01*x + px;
+      y  = 0.01*y + py;
+      px = x;
+      py = y;
+
+      // go forwards and turn according to the head rotation
+      if (y < 0.1) // ball far away, go quickly
+        mGaitManager->setXAmplitude(1.0 * xFactor);
```
Comment (fabienrohrer) [inline projects/contests/robotstadium/controllers/darwin-op_team_1/FieldPlayer.cpp:121]: Achtung, it's gaitManager rather than mGaitManager ^^

Please resolve the compilation issues before pushing.
Testing robotStadium seems also important. The robots should stand up as long as possible.
The 2 main differences between your samples and robotstadium are that the controllers runs asynchonously (cf: reference manual, Robot::synchronization) and the simulation basic time step is 40 (which is very high, provocating some instabilities to solve).

**Comment by DavidMansolino** [comment]
oh sorry, I did not really test it, I will do it.
